README.md

```diff
@@ -387,6 +391,19 @@ a line such as this to each job destination:
 ```
 <env file="/path/to/shared/galaxy/venv" />
 ```
+# Enable Galaxy to use BioContainers (Docker) <a name="auto-exec-tools-in-docker"/> [[toc]](#toc)
+This is a very cool feature where Galaxy automatically detects that your tool has an associated docker image, pulls it and runs it for you. These images (when available) have been generated using [mulled](https://github.com/mulled). To test, install the [iuc bedtools](https://toolshed.g2.bx.psu.edu/repository?repository_id=8d84903cc667dbe7&changeset_revision=7b3aaff0d78c) from the toolshed. When you try to execute *ClusterBed* for example. You may get a missing dependancy error for *bedtools*. But bedtools has an associated docker image on [quay.io](https://quay.io/).  Now configure Galaxy as follows:
+- Add this environment variable to `docker run`: `-e GALAXY_CONFIG_ENABLE_BETA_MULLED_CONTAINERS=True` 
+- In `job_conf.xml` configure a Docker enabled destination as follows:
+```
+<destination id="docker_local" runner="local">
+  <param id="docker_enabled">true</param>
+  <param id="docker_volumes">$galaxy_root:ro,$galaxy_root/database/tmp:rw,$tool_directory:ro,$job_directory:ro,$working_directory:rw,$default_file_path:rw</param>
+  <paraam id="docker_sudo">false</param>
+  <param id="docker_net">bridge</param>
```

Comment:
I don't know about enabling bridge like this - I'd discourage doing this or add a comment about why it is needed.